datasets/bangladesh-landcover-2001-2020.data.mdx

```diff
@@ -0,0 +1,60 @@
+---
+id: bangladesh-landcover-2001-2020
+name: 'Annual land cover maps for 2001 and 2020'
+description: 'Annual land cover maps for 2001 and 2020 (Bangladesh)'
+media:
+  src: ::file ./zz_placeholder.png
```

Comment:
Could use this image - https://unsplash.com/photos/d59NHNtT_Ss. 
We ideally want the cover images to be less than an MB. Here's a [guide](https://github.com/NASA-IMPACT/delta-config/blob/4b7d21655006a2d9056edd234ad46dd04dba69ee/docs/frontmatter/media.md#media) to how that can be achieved.
Don't forget to give proper attribution

```yaml
  author:
    name: Marek Piwnicki
    url: https://unsplash.com/photos/WiZOyYqzUss
```

Reply:
Done! Can you please check if it looks ok?